Pulumi: document how to install Telemetry backends (#742)
akka/akka-platform-guide: docs-source/docs/modules/deployment/pages/gcp-install-quickstart.adoc

== Expected Time to Install

> A new installation will take approximately  _20 minutes_, and by default, the following components will be provisioned:

Reviewer: Maybe `components will be provisioned` -> `steps will be executed`

Author: Done here and in the AWS version too. :-)

1. Create an GKE cluster
2. Install the Akka Cloud Platform Helm chart
3. Install the Grafana Helm chart
4. Install the Prometheus Helm Chart

> 5. Setup a PostgreSQL Database Instance

Reviewer 2: ```suggestion
5. Set up a PostgreSQL Database Instance
```

Just run `pulumi up` again, which is basically a noop, but the stack gets updated fully.

> NOTE: Another documented issue in Pulumi is a problem in concurrency control. In this case, an operation fails with the following message. This is also documented in https://www.pulumi.com/docs/troubleshooting/#conflict[Pulumi troubleshooting section {tab-icon}, window="tab"]

Reviewer 2: ```suggestion
NOTE: Another documented issue in Pulumi is within concurrency control. In this case, an operation fails with the following message. This is also documented in https://www.pulumi.com/docs/troubleshooting/#conflict[Pulumi troubleshooting section {tab-icon}, window="tab"]
```

----

> [#connect-grafana]

Reviewer: We should probably extract this fragment and include it in GCP and AWS instructions.

Author: Done!
